src/rabbit-publisher/rabbit-publisher.service.ts

```diff
@@ -59,45 +58,37 @@ export class RabbitPublisherService {
 
       await this.initializeRabbitMQ();
     } catch (error) {
-      console.error("Error connecting to RabbitMQ:", error);
+      logger.error('Error connecting to RabbitMQ:', error);
     }
   }
 
   async initializeRabbitMQ(): Promise<void> {
     try {
-      await this.channel.assertExchange(this.nameExchange, "direct", {
+      await this.channel.assertExchange(this.nameExchange, 'direct', {
         durable: false,
       });
       await this.channel.assertQueue(this.nameQueue, { durable: true });
-      await this.channel.bindQueue(
-        this.nameQueue,
-        this.nameExchange,
-        "message_type",
-      );
+      await this.channel.bindQueue(this.nameQueue, this.nameExchange, 'message_type');
     } catch (error) {
-      console.error("Error initializing RabbitMQ:", error);
+      logger.error('Error initializing RabbitMQ:', error);
     }
   }
 
   async publishMessageToCommunication(message: any): Promise<void> {
     if (!this.channel) {
-      console.error("Channel is not initialized");
+      logger.error('Channel is not initialized');
```

Comment:
למה לפעמים את עושה this.logger ולפעמים logger?

Reply:
שיניתי